### ISSUE TITLE
Split card's second half has `originalText` of the first 

### ISSUE THREAD

**Issue by mr-jonix** [mtgjson/mtgjson#63]
second half of a split card has `original text` of the first half (`text` is correct). Unless this was intentional, `originalText` should have the printed text of the second half.

Example (Concoct of Connive // Concoct):

```
{
            "artist": "Dan Scott",
            "borderColor": "black",
            "colorIdentity": [
                "B",
                "U"
            ],
            "convertedManaCost": 9.0,
            "foreignData": [],
            "frameVersion": "2015",
            "hasFoil": true,
            "hasNonFoil": true,
            "layout": "split",
            "legalities": {
                "1v1": "Legal",
                "brawl": "Legal",
                "commander": "Legal",
                "duel": "Legal",
                "frontier": "Legal",
                "future": "Legal",
                "legacy": "Legal",
                "modern": "Legal",
                "standard": "Legal",
                "vintage": "Legal"
            },
            "manaCost": "{3}{U}{B}",
            "multiverseId": 452972,
            "name": "Concoct",
            "names": [
                "Connive",
                "Concoct"
            ],
            "number": "222",
            "originalText": "Gain control of target creature with power 2 or less.",
            "originalType": "Sorcery",
            "printings": [
                "GRN",
                "PGRN"
            ],
            "rarity": "rare",
            "rulings": [
                {
                    "date": "2018-10-05",
                    "text": "Connive’s effect lasts indefinitely. It doesn’t wear off during the cleanup step, and it doesn’t expire if the creature’s power becomes greater after Connive has resolved."
                },
                {
                    "date": "2018-10-05",
                    "text": "The creature card you return with Concoct may be one that you just surveilled into your graveyard."
                },
                {
                    "date": "2018-10-05",
                    "text": "No player may take actions between the time you surveil and the time you return a creature card to the battlefield."
                },
                {
                    "date": "2018-10-05",
                    "text": "When you surveil, you may put all the cards you look at back on top of your library, you may put all of those cards into your graveyard, or you may put some of those cards on top and the rest of them into your graveyard."
                },
                {
                    "date": "2018-10-05",
                    "text": "You perform the actions stated on a card in sequence. For some spells and abilities, you’ll surveil last. For others, you’ll surveil and then perform other actions."
                }
            ],
            "subtypes": [],
            "supertypes": [],
            "text": "Surveil 3, then return a creature card from your graveyard to the battlefield.",
            "type": "Sorcery",
            "types": [
                "Sorcery"
            ],
            "uuid": "890ac54c-6fd7-4e46-8ce4-8926c6975f60"
        }
```

**Comment by ZeldaZach** [comment]
This is a known issue, but I went forward with the release anyway.

I need to figure out how to better manage scraping Gatherer, as this has been an issue for some time. Outside assistance would be awesome, otherwise I can figure it out myself for 4.1.

**Comment by ZeldaZach** [comment]
Fixed with #100
